### PR TITLE
Keycloak

### PR DESCRIPTION
**Pull request by gilleswittenberg**
- implement Keycloak
- clean up Grip

TODOs
- enable tests
- import Keycloak from npm vs. script tag
- SSI

### CODE REVIEW

File: src/app/state/auth/keycloak/keycloak.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const settings = {
+  // TODO: Switch between production and acceptance realm
+  "realm": "datapunt-ad-acc",
```
Comment (remcohoff) [inline src/app/state/auth/keycloak/keycloak.ts]: Can't we put this in the .env-file? So we can't forget to switch this.

File: src/app/state/rest/hooks/useApiRequest.test.tsx [hunk 2]
```diff
@@ -9,10 +9,9 @@ type Pet = {
   type: string
 }
 
-describe("useApiRequest", () => {
+xdescribe("useApiRequest", () => {
```
Comment (remcohoff) [inline src/app/state/rest/hooks/useApiRequest.test.tsx:12]: Typo?

File: src/app/state/auth/keycloak/keycloak.ts (hunk 1 above)
Comment (gilleswittenberg) [reply src/app/state/auth/keycloak/keycloak.ts]: Yes, have done that. Please refresh.

File: src/app/state/rest/hooks/useApiRequest.test.tsx (hunk 2 above)
Comment (gilleswittenberg) [reply src/app/state/rest/hooks/useApiRequest.test.tsx:12]: That's to (temporarily) disable the test suite. It's a TODO to fix this.

**Review by remcohoff — APPROVED** [review]

**Pull request merged**